js/medals.js

```diff
@@ -0,0 +1,116 @@
+function rettich_draw_medals(){
+    if (typeof ALL_STATS=='string'){
+        ALL_STATS = JSON.parse(ALL_STATS) // i still have no idea why i have to parse it here again
+    }
+
+
+    const ctx = document.getElementById('myChart');
+
+    let medal_labels = ALL_STATS[CURRENT_GROUP]['riders']
```

Comment:
Funktioniert das hier so? Bei mir kommt ein Fehler, dass CURRENT_GROUP noch nicht definiert sei (was auch stimmt)...

Reply:
Ja denke weil das noch Work in Progress ist. Ich würde auch sagen wenn du das ähnlich aufgeräumt hast dann musst du das auch nicht auf Teufel komm raus mergen sondern kannst einfach nur deine Version nehmen

Reply:
Ich habe das Problem bei mir gefunden. Liegt an globalen Variablen und Funktionen. Ich bin mir aber noch nicht sicher wie man das am besten löst. Weil wenn man das auf mehrere .js Dateien aufteilt, was meines Erachtens nach auch Sinn macht, muss man gucken wie man das mit den variablen und Funktionen macht, die in anderen Dateien definiert sind.